src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box } from '@mui/material';
+import React from 'react';
+import { VictoryBar, VictoryChart, VictoryAxis, VictoryTheme } from 'victory';
+
+import * as Styles from './styles';
+
+export default function CloroplethLegend() {
+  return (
+    <Styles.Container>
+      <svg style={{ height: 0 }}>
+        <defs>
+          <linearGradient id="myGradient">
+            <stop offset="25%" stopColor="#ADDC91" />
+            <stop offset="50%" stopColor="#6CC24A" />
+            <stop offset="75%" stopColor="#509E2F" />
+            <stop offset="100%" stopColor="#4A7729" />
+          </linearGradient>
+        </defs>
+      </svg>
+      <Box paddingTop="25px" px="25px">
```

Comment:
Add some spaces here to improve the readability, even this is good for code live

Reply:
Fixed!